ts/screens/wallet/TransactionsScreen.tsx

```diff
@@ -113,13 +116,24 @@ class TransactionsScreen extends React.Component<Props> {
       _ => _ === selectedWallet.idWallet
     );
 
+    const transactionsRefreshControl = (
+      <RefreshControl
+        onRefresh={() => {
+          this.props.loadTransactions();
+        }}
+        refreshing={false}
```

Comment:
I guess this is set to false because we want to show refresh control spinner only when user pulls down the list. Then it immediately disappears and the loading status is shown inside each single component (_TransactionsList_ in this case)

If it is so, could you leave a comment explaining this behavior?

Reply:
detailed within https://github.com/teamdigitale/io-app/pull/1307/commits/b9b02ef011e9449364aeeb0173201cec9bb26ed5